x-pack/plugins/aiops/public/components/log_rate_analysis/item_filter_popover.tsx

```diff
@@ -162,19 +175,21 @@ export const FieldFilterPopover: FC<FieldFilterPopoverProps> = ({
               <EuiButtonEmpty
                 size="xs"
                 flush="left"
-                onClick={() => setFieldsFilter(filteredUniqueFieldNames, true)}
-                disabled={fieldSearchText.length > 0 && filteredUniqueFieldNames.length === 0}
+                onClick={() => setItemsFilter(filteredUniqueItemNames, true)}
+                disabled={itemSearchText.length > 0 && filteredUniqueItemNames.length === 0}
                 data-test-subj="aiopsFieldSelectorSelectAllFieldsButton"
               >
-                {fieldSearchText.length > 0 ? (
+                {itemSearchText.length > 0 ? (
                   <FormattedMessage
                     id="xpack.aiops.logRateAnalysis.page.fieldSelector.selectAllSearchedFields"
-                    defaultMessage="Select filtered fields"
+                    defaultMessage="Select filtered {itemType}"
+                    values={{ itemType }}
                   />
                 ) : (
                   <FormattedMessage
                     id="xpack.aiops.logRateAnalysis.page.fieldSelector.selectAllFields"
-                    defaultMessage="Select all fields"
+                    defaultMessage="Select all {itemType}"
```

Comment:
The data grid columns control uses Show all / Hide all for these buttons, but I can see why you might want to use these labels as you are using the same component for the 'Filter fields' functionality and it may not be worth the effort to parameterize these labels.

If you do stick with the same text for the Columns and the Filter Fields controls, can you add a minimum width to the popover as currently the width of the popover changes if you add / remove search text?

Reply:
Updated in 72f120e9a39c579999d4b3aabcf548ff88d6fdf0